core/domain/stats_domain.py

```diff
@@ -45,38 +45,46 @@ class ExplorationStats(object):
     """Domain object representing analytics data for an exploration."""
 
     def __init__(
-            self, exp_id, exp_version, num_starts, num_actual_starts,
-            num_completions, state_stats_mapping):
+            self, exp_id, exp_version, num_starts_v1, num_starts_v2,
+            num_actual_starts_v1, num_actual_starts_v2, num_completions_v1,
+            num_completions_v2, state_stats_mapping):
         """Constructs an ExplorationStats domain object.
 
         Args:
             exp_id: str. ID of the exploration.
             exp_version: int. Version of the exploration.
-            num_starts: int. Number of learners who started the exploration.
-            num_actual_starts: int. Number of learners who actually attempted
-                the exploration. Theses are the learners who have completed
-                the initial state of the exploration and traversed to the next
-                state.
-            num_completions: int. Number of learners who completed the
-                exploration.
+            num_starts_v1, num_starts_v2: int. Number of learners who started
```

Comment:
Break into two separate lines and explain difference between v1 and v2. The second arg's description can be "As above, but for events with version 2." Ditto below.

Reply:
Done.